frontend/src/app/testQueue/AoEForm/AoEModalForm.jsx

```diff
@@ -85,68 +93,128 @@ const PriorTestInputs = ({
   setPriorTestResult,
   priorTestType,
   setPriorTestType,
+  mostRecentTest,
 }) => {
-  // disable inputs other than Yes/No if it is "Yes" or unset
-  const disableDetails = isFirstTest !== false;
+  const filledPriorTest =
+    (mostRecentTest.dateTested || "").split("T")[0] === priorTestDate &&
+    mostRecentTest.result === priorTestResult;
+  const [mostRecentTestAnswer, setMostRecentTestAnswer] = useState(
+    isFirstTest === undefined ? undefined : filledPriorTest ? "yes" : "no"
+  );
+  const previousTestEntry = (
+    <>
+      <DateInput
+        label="Date of Most Recent Test"
+        name="prior_test_date"
+        value={priorTestDate}
+        onChange={setPriorTestDate}
+        max={new Date().toISOString().split("T")[0]}
```

Comment:
I guess it's fine if `max` allows you to pick tomorrow even though it's still today?
For example for PST (notice date):
`new Date().toLocaleString()` => "2020-12-08T04:03:54.668Z"
`new Date().toISOString()` => "12/7/2020, 8:03:55 PM"

Versus
"12/7/2020, 8:03:35 PM"
"2020-12-08T04:03:36.809Z"